app/javascript/components/record-form/form/subforms/subform-dialog-fields/component.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Why do you need to disable this? This code only handles the subform fields, to make this work at the form level you will need to modify the file:

`app/javascript/components/record-form/components/render-form-sections.js`